beetsplug/edit.py

```diff
@@ -324,3 +333,55 @@ def save_changes(self, objs):
             if ob._dirty:
                 self._log.debug('saving changes to {}', ob)
                 ob.try_sync(ui.should_write(), ui.should_move())
+
+    # Methods for interactive importer execution.
+
+    def before_choose_candidate_listener(self, session, task):
+        """Append an "Edit" choice and an "edit Candidates" choice (if
+        there are candidates) to the interactive importer prompt.
+        """
+        choices = [PromptChoice('d', 'eDit', self.importer_edit)]
+        if task.candidates:
```

Comment:
Ah, good point. I also don't see an immediate solution, so let's revisit the problem later on.